tensorrt_llm/_torch/pyexecutor/model_loader.py

```
@@ -160,6 +160,7 @@ def __init__(self,
                  pytorch_backend_config: PyTorchConfig,
                  mapping: Mapping,
                  spec_config: Optional["DecodingBaseConfig"],
+                 sparse_attention_config: Optional["SparseAttentionConfig"],
```

Ruff (0.13.1)

Undefined name `SparseAttentionConfig`

(F821)